tests/Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot

```diff
@@ -132,20 +132,24 @@ Spawn Notebook With Arguments
    [Documentation]  Selects required settings and spawns a notebook pod. If it fails due to timeout or other issue
    ...              It will try again ${retries} times (Default: 1). Environment variables can be passed in as kwargs
    ...              By creating a dictionary beforehand, e.g. &{test-dict}  Create Dictionary  name=robot  password=secret
-   [Arguments]  ${retries}=1  ${image}=s2i-generic-data-science-notebook  ${size}=Small  ${spawner_timeout}=600 seconds  &{envs}
+   [Arguments]  ${retries}=1  ${image}=s2i-generic-data-science-notebook  ${size}=Small  ${spawner_timeout}=600 seconds  ${refresh}=${False}  &{envs}
    FOR  ${index}  IN RANGE  0  1+${retries}
       ${spawner_ready} =    Run Keyword and Return Status    Wait Until JupyterHub Spawner Is Ready
       IF  ${spawner_ready}==True
          Select Notebook Image  ${image}
          Select Container Size  ${size}
+         IF   ${refresh}
```

Comment:
why refreshing?

Reply:
we want to reload page after selecting the container size to check if after selecting it is it going back to default or getting modified

Reply:
to test that the size selection is not overwritten by the spawner I think